docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_home_page_translations.yml

```diff
@@ -0,0 +1,75 @@
+id: yukon_migrate_home_page_translations
+label: Yukon Migrate Home Page Node Translations
+audit: true
+migration_tags:
+    - Drupal 7
+    - node
+    - Home Page
+migration_group: legacy
+source:
+    plugin: d7_node_entity_translation
+    node_type: homepage
+    constants:
+        default_text_format: full_html
+destination:
+    plugin: entity:node
+    translations: true
+    destination_module: content_translation
```

Comment:
add bundle here
default_bundle: homepage

Reply:
This is resolved.